tests/test_lib.py

```diff
@@ -0,0 +1,5 @@
+import pytest
+
+
+def test_hello_worl():
```

Comment:
😉 
```suggestion
def test_hello_world():
```

Reply:
I added this one to see if you were awake. ☕️ ;)